packages/form/src/hooks/useOnFieldChange.ts

```diff
@@ -8,8 +8,8 @@ type CallbackFn<FieldValue, AllValues> = (
 
 export const useOnFieldChange = <FieldValue = unknown, AllValues = unknown>(
   name: string,
-  condition: boolean,
   callback: CallbackFn<FieldValue, AllValues>,
+  condition = true,
```

Comment:
It's maybe better to name this `enabled` ?

Reply:
right !